implementation/test/DepositFraudTest.js

```diff
@@ -62,9 +43,11 @@ describe("DepositFraud", async function() {
 
   describe("provideFundingECDSAFraudProof", async () => {
     let timer
+    let bond
 
     before(async () => {
       timer = await tbtcConstants.getFundingTimeout.call()
+      bond = 1000000
```

Comment:
Why not make `bond` a `const` above and initialize it outside of `before`?

Reply:
changed it, no reason I can think of other than "the other one was like that" 😄